src/StringManipulation/VMSolutionOne.java

```diff
@@ -0,0 +1,21 @@
+class Solution {
+    public boolean backspaceCompare(String S, String T) {
+        String sFinal = finalString(S);
```

Comment:
This means you are using O(n+m) space

Reply:
yeah correct. From this came up with O(1)